package.json

```diff
@@ -9,6 +9,8 @@
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
+    "chart.js": "^3.7.1",
+    "ng2-charts": "^3.0.8",
     "react": "^17.0.2",
```

Comment:
These packages are not required, as react-chartjs-2 package is used. Can you please remove these packages